### PR TITLE
Add Minimal example

### PR DESCRIPTION
**Pull request by LuckyPigeon**
refer to #86 
@dai-shi 
Add another example.
Work both in local environment and codesandbox.

### CODE REVIEW

**Comment by dai-shi** [comment]
@LuckyPigeon 
Hey, thanks for another example. Here're some notes:

- I'd prefer `examples/minimal` instead of `examples/Minimal`.
- Anyway, I found `minimal` not very descriptive. That was my bad in the md file.
- If you run `examples/hello` locally, it'd use the local src of jotai. I'm not sure how useful this is, but we'd want to make it consistent. See `examples/hello` how it's using customize-cra in package.json, `config-overrides.js` and `.env.development`.
- I'm not sure if we need tsconfig.json or not, but want to keep consistent throughout all examples.

**Comment by LuckyPigeon** [comment]
@dai-shi 
I updated the example name to text length counter, using `customize-cra` and build it with `yarn`.

**Comment by dai-shi** [comment]
@LuckyPigeon Thanks so much!
Let's use `_` instead of a space.
Making it a little shorter, how about `examples/text_length` ?

File: examples/text_length/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Minimal Example
+
+```bash
+git clone https://github.com/pmndrs/jotai
+
+# install project dependencies & build the library
+cd jotai && yarn
+
+# move to the examples folder & install dependencies
+cd examples/Minimal && yarn
```
Comment (dai-shi) [inline examples/text_length/README.md:10]:
```suggestion
cd examples/text_length && yarn
```
Not sure, how we can effectively maintain this file...

**Comment by dai-shi** [comment]
https://githubbox.com/LuckyPigeon/jotai/tree/add_examples/examples/text_length
Confirmed it's working as expected.

**Comment by dai-shi** [comment]
Merging this. We can keep updating to improve examples and keep consistency.
Thanks for your contribution! Please keep it up!

**Pull request merged**